### ISSUE TITLE
objects have no __repr__

### ISSUE THREAD

**Issue by Neon22** [pyscript/ltk#8]
When printing jquery objects to discover what they are and what can be done witth them:
you generally just get `[object object]`

Is there a way to add `__repr__` to jquery classes (or somesuch mechanism) so we can see what they are ?

**Comment by laffra** [comment]
That is correct. On MicroPython, you get something like `<JsProxy 403>` instead of `[object Object]`. This indicates that the PyScript FFI does not know what to do with the JavaScript object when converting it to a Python string. 

When you add the following JavaScript source to your `index.html`:
```
$.fn.toString = function() {
  return this.html();
};
```
You will get better print statements, saying something like `jQuery[BUTTON, "Click Me"]`

**Comment by laffra** [comment]
You can also try `outerHTML()` or `text()` instead of `html()`.

**Comment by laffra** [comment]
I used the outerHTML version and added it to LTK v0.2.30. It now prints something like:

```
jQuery[BUTTON, "<button class="ltk-button" ltk_id="11959584" style="margin-top: 24px; border-radius: 8px; padding: 12px;">Increment Product 1 count</button>"]
```
